src/QsCompiler/Compiler/CompilationLoader.cs

```diff
@@ -71,6 +71,11 @@ public struct Configuration
             /// </summary>
             public bool SkipMonomorphization;
             /// <summary>
+            /// If set to true, the compiler will include all environment-defined callables and types
+            /// in the resulting syntax tree.
+            /// </summary>
+            public bool DoIntrinsicMapping;
```

Comment:
I would actually suggest to hold off with making this configurable in the same way that other things are (meaning I would actually not yet write the handle in the compilation loader - it's quick to add after all). It is not yet clear to me whether that makes sense or not.